lib/methods/ifwapichecktask.cpp

```diff
@@ -344,6 +249,31 @@ void IfwApiCheckTask::ScriptFunc(const Checkable::Ptr& checkable, const CheckRes
 	String username = resolveMacros("$ifw_api_username$");
 	String password = resolveMacros("$ifw_api_password$");
 
+	std::function<void(const Value& cmdLine, const ProcessResult&)> resultHandler;
+	if (Checkable::ExecuteCommandProcessFinishedHandler) {
+		auto callback = Checkable::ExecuteCommandProcessFinishedHandler;
+		resultHandler = [cr, callback = std::move(callback)](const Value& cmdLine, const ProcessResult&) {
```

Comment:
I already wondered with previous revisions why you define a variable for then to just move it into the lambda capture instead of just doing `callback = Checkable::ExecuteCommandProcessFinishedHandler` there.

Though, other variant that would avoid repeating that long name:
```suggestion
	if (auto callback = Checkable::ExecuteCommandProcessFinishedHandler; callback) {
		resultHandler = [cr, callback = std::move(callback)](const Value& cmdLine, const ProcessResult&) {
```